### PR TITLE
Use Redux in React components

### PR DESCRIPTION
**Pull request by djo1975**
- Connected store to the main component of React app.
- Defined a state slice which contain  book
- Displayed books, received from the slice, in a reusable component
- Dispatched actions using useDispatch
- Added a <Button> component, which includes:
      An event handler that adds a book to the books array (with attributes id, title and author)
- Added a <Button> component, which includes:
      An event handler that removes a book from the books array (by id)
- Added a <Button> component, which includes:
     An event handler that check status (i.e add a functionality for "Check status" button click)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +24,11 @@
...
+# 📖Bookstore: Use Redux in React components<a name="about-project"></a>
 
-**Bookstore: add reducers and actions**
+**Bookstore: Use Redux in React components**
 
-In this step, you will implement Redux Toolkit inside of a React application. You will:
-
-- Setup a Redux store
-- Create a slice to save your state and reducers
+In this step you will use a Redux store to display books. You'll implement react-redux in order to connect your store and then access your state variables (accessed through useSelector) and state modifiers (which will be a combination of a reducer and useDispatch).
```
Comment (rloterh) [inline README.md:32]: - [ ] Live demo link on _line 52_ of your ReadMe is broken, kindly endeavor to fix or remove it if this app is not yet deployed. Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mladan, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦   🏃 

## Highlights 👍  
- Redux state management properly applied to React components!
- ReadMe is near professional!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (djo1975) [reply README.md:32]: Thank you for helping me make the project more professional. I'm confident that everything is now in line with your suggestions

**Review by addod19 — APPROVED** [review]
### Hi @djo1975, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] The README file is descriptive :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by djo1975** [comment]
Thanks guys

**Pull request merged**